Error: The `form` element was not requested.
Error: Use `aria-describedby` to reference a group description.
Error: For usability, it is recommended to put the group description at the top of the component, just after the label.
Error: For broadest accessibility, implicitly associated `label` elements should also be explicitly associated with their control, using `for`.
Error: Checkboxes in a group should use a `name` value that ends with "[]" to allow server-side code to collect the values as an array (or similar).
Error: `aria-invalid="true"` is used to accessibly indicate fields that have errors.
Error: When a checkbox is marked as `required`, it must be checked to be valid. Per the instructions, the user only needs to choose one or more, so `required` must not be used on any of the checkbox inputs.
Error: Color alone should not be used to convey errors.
Error: The error text should not be hidden when the component has an error.
Error: styles were not requested. Note: If checkbox elements are wrapped in their `label`, `br` is unnecessary.
diff --git a/output/Checkbox Group with Validation Error/3/56e34faa-c475-43dd-ac9d-6032655973fa.html b/output/Checkbox Group with Validation Error/3/56e34faa-c475-43dd-ac9d-6032655973fa.html
index 4b6dfa1..25de183 100644
--- a/output/Checkbox Group with Validation Error/3/56e34faa-c475-43dd-ac9d-6032655973fa.html	
+++ b/output/Checkbox Group with Validation Error/3/56e34faa-c475-43dd-ac9d-6032655973fa.html	
@@ -1,18 +1,16 @@
-<form>
-  <fieldset>
-    <legend>What fruits do you like?</legend>
-    <label><input type="checkbox" name="fruits" value="bananas" required> Bananas</label><br>
-    <label><input type="checkbox" name="fruits" value="oranges"> Oranges</label><br>
-    <label><input type="checkbox" name="fruits" value="apples"> Apples</label><br>
-    <div class="error-message">You must choose one or more fruits</div>
-  </fieldset>
-</form>
-<style>
-  .error-message {
-    display: none;
-    color: red;
-  }
-  input:invalid ~ .error-message {
-    display: block;
-  }
-</style>
\ No newline at end of file
+<fieldset aria-describedby="fruits-error">
+  <legend>What fruits do you like?</legend>
+  <div class="error-message" id="fruits-error">You must choose one or more fruits</div>
+  <label for="bananas">
+    <input type="checkbox" name="fruits[]" id="bananas" value="bananas" aria-invalid="true">
+    Bananas
+  </label>
+  <label for="oranges">
+    <input type="checkbox" name="fruits[]" id="oranges" value="oranges" aria-invalid="true">
+    Oranges
+  </label>
+  <label for="apples">
+    <input type="checkbox" name="fruits[]" id="apples" value="apples" aria-invalid="true">
+    Apples
+  </label>
+</fieldset>
